source/_components/sensor.teksavvy.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "TekSavvy"
+description: "Instructions how to integrate TekSavvy data usage within Home Assistant."
+date: 2017-12-17 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: teksavvy.png
+ha_category: Sensor
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+Integrate your [TekSavvy](https://myaccount.teksavvy.com/) account information into Home Assistant.
+
+You can get your API key from 
+https://myaccount.teksavvy.com/ApiKey/ApiKeyManagement
+
+To use your TekSavvy sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: teksavvy
+    api_key: API_KEY
+    total_bandwidth: 400
+    monitored_variables:
+     - usage
+     - usage_gb
+     - limit
+     - onpeak_download
+     - onpeak_upload
+     - onpeak_total
+     - offpeak_download
+     - offpeak_upload
+     - offpeak_total
+     - onpeak_remaining
+```
+
+Configuration variables:
```

Comment:
@jpjodoin Please replace this part with the configuration tags.
For more information about using those, please see: https://home-assistant.io/developers/documentation/create_page/#configuration

Reply:
Perfect, I will change that. Quick question, right now I'm using the configuration "monitored_variables", do you think it would be preferable if I would change for "monitored_conditions" ? Is this a standard configuration name ?

Thanks !

Reply:
I'm not sure, that question would be more appropriate in the parent PR.

Reply:
Ok, actually both are part of the standard const, I will keep monitored_variables as it seems more appropriate. Thanks.